test/integration/typescript/pages/routerEvents.tsx

```diff
@@ -0,0 +1,33 @@
+import React, { useEffect, useState } from 'react'
+import Router from 'next/router'
+
+const RouterEvents: React.FC = () => {
+  const [routerEvent, setRouterEvent] = useState('')
+  const [routerEventError, setRouterEventError] = useState('')
+
+  useEffect(() => {
+    Router.events.on('routeChangeStart', () =>
+      setRouterEvent('routeChangeStart')
+    )
+
+    Router.events.on('routeChangeComplete', () =>
+      setRouterEvent('routeChangeComplete')
+    )
+
+    Router.events.on('routeChangeError', (error) => {
+      setRouterEvent('routeChangeError')
+
+      setRouterEventError(error.message)
+    })
```

Comment:
Looks like this test is useless as the events don't require to import the types 🤣 . Which is great btw so probably we don't need it.

Reply:
Hahahah yeah. I'll remove it then